mod_ci/controllers.py

```diff
@@ -822,14 +822,17 @@ def progress_reporter(test_id, token):
                     log.debug('Test {id} completed: {crashes} crashes, {results} results'.format(
                         id=test.id, crashes=crashes, results=results
                     ))
+                    crash = True
```

Comment:
You can derive this from the state variable as well... This seems a bit unnecessary